tools/mo/openvino/tools/mo/main.py

```diff
@@ -364,7 +364,8 @@ def check_fallback(argv : argparse.Namespace):
 
     fallback_reasons['extensions'] = legacy_extensions_used
     fallback_reasons['transformations_config'] = legacy_transformations_config_used
-    fallback_reasons['input_freezing'] = input_freezig_used
+    if argv.framework != "onnx":
```

Comment:
do we have such limitation for other frontends? We don't for TF FE, what is this about PDPD FE?
@ceciliapeng2011, can you please comment on this on behalf of PDPD FE?

Reply:
@bszmelcz, please remove this check at all because TF FE has no such limitations and PDPD has no legacy frontend for the fallback.